src/chapters/setup-intro-data-types/studio.rst

```diff
@@ -1,2 +1,68 @@
 Studio: Data Types
 ===================
+
```

Comment:
Carly, this looks good for a first studio.  However, it seems really short.  Perhaps we could add a second task that requires a second file?